fixer_suites/three_to_two/tests/test_reload.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+
+@pytest.fixture(name="fixer")
+def fixer_fixture(three_to_two_test_case):
+    return three_to_two_test_case("reload")
+
+
+def test(fixer):
```

Comment:
Could you rename this to be more explicit about what is being tested?

Reply:
Renamed 👍